backend/amd64/emit.ml

```diff
@@ -1442,13 +1561,25 @@ let emit_instr ~first ~fallthrough i =
     | _ ->
       let lbl = add_float_constant f in
       I.movsd (mem64_rip REAL8 (L.encode lbl)) (res i 0))
-  | Lop (Const_vec128 { high; low }) -> (
-    match high, low with
+  | Lop (Const_vec128 { word0; word1 }) -> (
+    match word0, word1 with
```

Comment:
not matter, but it feels likw
`word1; word0` might be
slightly more natural?)

Reply:
Done